src/status_im/data_store/realm/core.cljs

```diff
@@ -104,11 +124,12 @@
 (defn- migrate-schemas
   "Apply migrations in sequence and open database with the last schema"
   [file-name schemas encryption-key current-version]
-  (log/info "migrate schemas")
-  (doseq [schema schemas
-          :when (> (:schemaVersion schema) current-version)
-          :let [migrated-realm (open-realm schema file-name encryption-key)]]
-    (close migrated-realm))
+  (log/info "migrate schemas" current-version)
+  (when (pos? current-version)
```

Comment:
Is it ok not to migrate on newly created accounts? I take yes, but I would double check

Reply:
Yes, i tested and it works fine. No need to iterate migrations in order to create a new database.